src/pages/Admin/AdminVenueRoomDetails.tsx

```diff
@@ -115,19 +117,14 @@ export const AdminVenueRoomDetails = ({
                   }
                 </div>
                 <div className="toggle-room">
-                  <label id={"toggle-" + index} className="switch">
-                    <input
-                      type="checkbox"
-                      id={"toggle-" + index}
-                      name={"toggle-" + index}
-                      checked={room.isEnabled}
-                      onClick={() => {
-                        updateRoom(!room.isEnabled);
-                      }}
-                    />
-                    <span className="slider round"></span>
-                  </label>
-                  <div>Turn room {room.isEnabled ? "Off" : "On"}</div>
+                  <Toggler
+                    id={"toggle-" + index}
+                    checked={room.isEnabled}
+                    onToggle={() => {
+                      updateRoom(!room.isEnabled);
+                    }}
+                    label={room.isEnabled ? "Turn room Off" : "Turn room On"}
+                  />
```

Comment:
There's no need for an `id`, because `Checkbox` nests it's `input` within the `label`, so they are inherently connected without needing the `id`.

```suggestion
                  <Toggler
                    checked={room.isEnabled}
                    onChange={() => {
                      updateRoom(!room.isEnabled);
                    }}
                    label={room.isEnabled ? "Turn room Off" : "Turn room On"}
                  />
```

The `onChange` here should be `useCallback`d

Reply:
removed `id` in a64982d7f2094afc0571f671312f576d4007dbcb